### ISSUE TITLE
Scheduled background sync of activity data

### ISSUE THREAD

**Issue by envolyse** [Freeyourgadget/Gadgetbridge#242]
It would be a good option for a usecase I described [here](https://github.com/Freeyourgadget/Gadgetbridge/issues/241) as it would be totally automatic.

**Comment by danielegobbetti** [comment]
This is a task that could be interesting for a new contributor, as it doesn't need to interact with devices etc, but it's rather a "pure" Android development take.

Would someone step up and develop this? Thanks!

**Comment by envolyse** [comment]
I was just looking for ways to implement this and realised that only Mi Band has 'Sync' function. So manually pulling pebble's activity data from phone's side is not possible or do I miss something?

**Comment by danielegobbetti** [comment]
Correct, at least for pebble health: the data are exchanged using datalog messages, and these messages are sent spontaneously by the watch.
There are some apps that allow you to trigger this behaviour, but as far as I know they are all watchapps, so nothing we can do on the phone side.

Datalog transmitted data should not be older than 15 minutes, though.
